rsmtool/reporter.py

```diff
@@ -317,11 +319,21 @@ def convert_ipynb_to_html(notebook_file, html_file):
         :ref:`render_notebook <render_notebook>` command-line utility.
         """
 
+        # `nbconvert` uses `asyncio` which uses an entirely default
```

Comment:
Would Azure be sufficient for testing this? (Not that we can do much about it)

Reply:
Yes, the Azure build was failing without this which is how I got turned on to this issue.